plugin/entryBase.go

```diff
@@ -99,6 +99,12 @@ func (e *EntryBase) Name() string {
 	return e.name
 }
 
+// ID returns the entry's ID. It won't panic on an empty
```

Comment:
Yeah, this makes sense. Do you think plugin authors would find this useful? If not, may be worth telling them to ignore this.

Reply:
Probably don't need to use it. We use it only to determine the plugin's name, which isn't a general problem. I guess we should either clarify what ID is (I'd just point to the description of the ID wrapper method) or note that it's primarily for implementing the external plugin support. I think I'll do the latter.